### PR TITLE
add lesson1.4

### PR DESCRIPTION
**Pull request by Tanyaa-a**

### CODE REVIEW

File: src/AddTodoForm.jsx [hunk 1]
```diff
@@ -1,20 +1,33 @@
+import React from "react";
+
 const TodoForm = (props) => {
```
Comment (ethan-carlson) [inline src/AddTodoForm.jsx]: Destructure props

File: src/AddTodoForm.jsx [hunk 2]
```diff
@@ -1,20 +1,33 @@
...
-            <button type="submit">Add</button>
-        </form>
-    )
-}
-export default TodoForm
\ No newline at end of file
+  const [todoTitle, setTodoTitle] = React.useState("");
+  const handleChange = (event) => {
+    console.log(event.target.value);
+  };
```
Comment (ethan-carlson) [inline src/AddTodoForm.jsx]: remove unused function

File: src/AddTodoForm.jsx [hunk 3]
```diff
@@ -1,20 +1,33 @@
...
+  const handleTitleChange = (event) => {
+    let newTodoTitle = event.target.value;
+    setTodoTitle(newTodoTitle);
+  };
+  const handleAddTodo = (event) => {
+    event.preventDefault();
+    //let todoTitle = event.target.title.value; //retrieve the value of the title element from the event target and store it in a variable named todoTitle
+    // event.target.reset();
```
Comment (ethan-carlson) [inline src/AddTodoForm.jsx]: remove unused code

**Review by ethan-carlson — APPROVED** [review]
Nice work!

You missed destructuring one `props` in `AddTodoForm`, but otherwise you meet the requirements and your code is nicely written.

Just a reminder to not check in code that doesn't do anything whenever possible.  if you want to record something for reference, you should include a comment why you want to keep it in.  It's important to keep your main branch clean.

File: src/AddTodoForm.jsx (hunk 1 above)
Comment (Tanyaa-a) [reply src/AddTodoForm.jsx]: Thank you Ethan! Just corrected all the сomments

**Pull request merged**